frontend/appflowy_flutter/packages/flowy_infra/lib/colorscheme/lemonade.dart

```diff
@@ -59,7 +59,7 @@ class LemonadeColorScheme extends FlowyColorScheme {
           main1: _lightDandelionYellow,
           // cursor color
           main2: _lightDandelionYellow,
-          shadow: _black,
+          shadow: const Color.fromRGBO(0, 0, 0, 0.15),
```

Comment:
I wonder if this would be the same as this:
```suggestion
          shadow: const Color(0x26000000),
```
Just curious. 

<details>
  <summary>How to calculate alpha?</summary>
```
FF = 255
15% of 255 = (255 * 0.15) = 38,25
Convert ~38.25 (38) to Hexadecimal = 26
```
</details>

Reply:
Yes, they are the same. I feel it is easier to understand opacity in `fromRGBO` way. 

Here is a link that lists the common opacity in Hex way if you are interested. 
https://gist.github.com/passiondroid/51fd6b3281c10d463361fa5eb5a1f22b